v-next/example-project/hardhat.config.ts

```diff
@@ -126,6 +128,15 @@ const pluginExample = {
   ],
 };
 
+// Type includes the builtin global options
```

Comment:
Is this meant to be a type-level tests? If so, let's place it in `hardhat`, as here it may go away accidentally.

Reply:
yeah it was the first thing I did to test the augmentation works. But then I added a type test, so we can probably remove it.

Reply:
Done in https://github.com/NomicFoundation/hardhat/pull/5494/commits/1deda23713471da2ef4a85a4ef4c22e42b4d959b